lib/iris/tests/unit/experimental/ugrid/load/test_load_mesh.py

```diff
@@ -24,7 +24,8 @@ def test_calls_load_meshes(self):
         args = [("file_1", "file_2"), "my_var_name"]
         with PARSE_UGRID_ON_LOAD.context():
             _ = load_mesh(args)
-        self.assertTrue(self.load_meshes_mock.called_with(args))
+        assert self.load_meshes_mock.call_count == 1
+        assert self.load_meshes_mock.call_args == ((args, None),)
```

Comment:
The `called_with` isn't playing ball in `py312` for some reason.

Didn't investigate any deeper, instead used an alternative pattern.

Reply:
I don’t get how that ever worked 😕 there is `assert_called_with` or `assert_called_once_with`
https://docs.python.org/3.12/library/unittest.mock.html#unittest.mock.Mock.assert_called_with

Reply:
Me either 🤷